src/facet/selection/_parameters.py

```diff
@@ -243,7 +244,8 @@ def _to_expression(self, path_prefix: Union[str, List[str]]) -> Expression:
 
         def _values_to_expression(values: ParameterSet) -> Expression:
             if isinstance(values, rv_frozen):
-                values: rv_frozen
+                # mypy - disabling due to lack of support for dynamic types
+                values = cast(rv_frozen, values)  # type: ignore
```

Comment:
Remove these two lines: the "is instance" already guarantees that values is an instance of `rv_frozen`. The additional cast is redundant.

Reply:
Sure, removed